[jit] Exported a method to get the supported list of elementwise ops

navahgar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #59581 [nnc] Added micro-benchmark to show perf improvement with cat subgraph optimization
* #59580 [nnc]  Added a transformation to move users of aten::cat to its inputs in the NNC kernel subgraph.
* **#60162 [jit] Exported a method to get the supported list of elementwise ops**
* #59579 [jit] Made a list for element-wise ops.

Differential Revision: [D29190841](https://our.internmc.facebook.com/intern/diff/D29190841)

navahgar committed:
[jit] Exported a method to get the supported list of elementwise ops

[ghstack-poisoned]

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29190841).

bertmaher commented on torch/csrc/jit/passes/tensorexpr_fuser.h:
```diff
@@ -60,6 +60,7 @@ TORCH_API bool usedOnlyInSize(Value* v);
 
 namespace tensorexpr {
+TORCH_API const OperatorSet& supported_eltwise_set();
 TORCH_API bool isSupported(Node* node);
```
While we're here let's fix the naming conventions so that we consistently use lowerCamelCase.

navahgar committed:
Update on "[jit] Exported a method to get the supported list of elementwise ops"


Differential Revision: [D29190841](https://our.internmc.facebook.com/intern/diff/D29190841)

[ghstack-poisoned]

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29190841).

navahgar committed:
Update on "[jit] Exported a method to get the supported list of elementwise ops"


Differential Revision: [D29190841](https://our.internmc.facebook.com/intern/diff/D29190841)

[ghstack-poisoned]

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29190841).

navahgar committed:
Update on "[jit] Exported a method to get the supported list of elementwise ops"


Differential Revision: [D29190841](https://our.internmc.facebook.com/intern/diff/D29190841)

[ghstack-poisoned]

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29190841).
